vendor/github.com/ceph/ceph-csi/api/deploy/kubernetes/csi-config-map.go

```diff
@@ -46,8 +46,6 @@ type RBD struct {
 	NetNamespaceFilePath string `json:"netNamespaceFilePath"`
 	// RadosNamespace is a rados namespace in the pool
 	RadosNamespace string `json:"radosNamespace"`
```

Comment:
This should not be undone, it is part of `api/deploy/kubernetes/...` :exploding_head: